.ci/build-mac.sh

```diff
@@ -1,28 +1,24 @@
 #!/bin/sh -ex
-
 brew install -f --overwrite nasm ninja git p7zip create-dmg ccache pipenv
-
 #/usr/sbin/softwareupdate --install-rosetta --agree-to-license
 arch -x86_64 /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
-arch -x86_64 /usr/local/homebrew/bin/brew update
-arch -x86_64 /usr/local/homebrew/bin/brew install -f --overwrite llvm@16 sdl2 glew cmake
-arch -x86_64 /usr/local/homebrew/bin/brew link -f llvm@16
-
```

Comment:
These line removals are unnecessary and make the file less readable imo.

Reply:
Homebrew complains about the path then builds from source thats why it’s necessary